### DISCUSSION TITLE
Telegram silent notifications

### DISCUSSION THREAD

**Discussion by fratzika** [0x2142/frigate-notify#168] [General]
Hello,

Are there any settings to get telegram notifications silently?
Example case:
If a person(label) is detected then send it normally else send it silently

The function for the telegram bot to send silently is disable_notification: true.
If this does not exist, can it be added?

Thanks in advance!

**Comment by freefd** [comment]
For the context: 
* It's about `disable_notification` for the BaseChat: https://github.com/go-telegram-bot-api/telegram-bot-api/blob/master/configs.go#L281
* API reference: https://core.telegram.org/bots/api#sendmessage

Frankly speaking, this may be treat as a notifier-specific gray area: what capabilities of a particular notifier can be tied to label filtering.
For example, I see the following capabilities that could be covered by this functionality for Telegram notifier:
* disable_notification
* protect_content

**Comment by 0x2142** [comment]
Hi @fratzika - At the moment this is not supported. I'll take a look & add it to my list 👍
